VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_

BS_:

BU_:


BO_ 2147569665 GPS_Maus_2: 8 Vector__XXX
 SG_ Altitude : 40|16@1- (1,0) [0|0] "" Vector__XXX
 SG_ N_Sat : 32|8@1- (1,0) [0|0] "" Vector__XXX
 SG_ UTC : 0|32@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2147569664 GPS_Maus_1: 8 Vector__XXX
 SG_ W_E : 63|1@1- (1,0) [0|0] "" Vector__XXX
 SG_ Longitude : 32|31@1+ (0.0001,0) [0|0] "" Vector__XXX
 SG_ N_S : 31|1@1- (1,0) [0|0] "" Vector__XXX
 SG_ Latitude : 0|31@1+ (0.0001,0) [0|0] "" Vector__XXX



BA_DEF_  "BusType" STRING ;
BA_DEF_DEF_  "BusType" "CAN";
